caffe2/core/nomnigraph/CMakeLists.txt

```diff
@@ -2,9 +2,11 @@
 file(GLOB_RECURSE NOMNI_SRCS *.cc)
 file(GLOB_RECURSE NOMNI_TEST_SRCS *Test.cc)
 exclude(NOMNI_SRCS "${NOMNI_SRCS}" "${NOMNI_TEST_SRCS}")
+set(NOMNI_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
 list(APPEND Caffe2_CPU_SRCS ${NOMNI_SRCS})
-list(APPEND Caffe2_CPU_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/include)
+list(APPEND Caffe2_CPU_INCLUDE ${NOMNI_INCLUDE_DIR})
+list(APPEND Caffe2_GPU_INCLUDE ${NOMNI_INCLUDE_DIR})
```

Comment:
why are these 2 identical lines needed?

Reply:
Line 8 adds folder to CPU includes, while line 9 to GPU includes.
And include lists are not shared between `torch_cpu` and `torch_gpu`

Reply:
I'm blind, sorry